web/src/components/tasks/business-formation/BusinessFormationPaginator.tsx

```diff
@@ -407,6 +407,10 @@ export const BusinessFormationPaginator = (props: Props): ReactElement => {
     return !allApiErrorsHaveMappings;
   };
 
+  const hasMissingSignature = (): boolean => {
+    return !!state.formationFormData.signers?.find((signer) => !signer.signature);
+  };
+
```

Comment:
[pebble] This error check feels like it should probably be part of the `useFormationErrors` hook. Is there a reason we want this to be closer to component? If we move it out we can probably pick up the error in the `allCurrentErrorsForStep` function. This might allow you to take out the `additionalErrors` prop from `FieldEntryAlert` and just render it with the existing map.

Reply:
Soma and I talked about this. This is the only error that we have where the field name is not also the error messaged displayed in the alert. `useFormationErrors` is probably the right place for this logic, but because this is a one off and `useFormationErrors` is pretty generic fitting this in would break a bunch of assumptions we have in that logic. We're going to keep this as is for now. If this is a pattern that we will use in other places, we'll need to make changes to `useFormationErrors`.

Additionally, Soma is following up with the design team to make sure that this is still a pattern we want to pursue. This ticket is from October 2022. If so, should this error message also hyperlink to the Signers field?